src/homework17/Main.java

```diff
@@ -0,0 +1,81 @@
+package homework17;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+        PetrolStation ps = new PetrolStation(1000);
+        Thread thread1 = new Thread(){
+            public void run(){
+                try {
+                    ps.doRefuel(50);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        };
+        Thread thread2 = new Thread(){
+            public void run(){
+                try {
+                    ps.doRefuel(88);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        };
+        Thread thread3 = new Thread(){
+            public void run(){
+                try {
+                    ps.doRefuel(25);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        };
+        Thread thread4 = new Thread(){
+            public void run(){
+                try {
+                    ps.doRefuel(10);
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                }
+            }
+        };
+
+        //Implementing a queue in real Life
+        thread1.start(); Thread.sleep(10);
+        thread2.start(); Thread.sleep(10);
+        thread3.start(); Thread.sleep(10);
+        thread4.start();
```

Comment:
what is happening here:
1. thread1 starts new thread
2. current thread (Main) goes to sleep for a 10 seconds
3. thread2 starts new thread
4. etc.

this is incorrect behaviour, sleep must be inside `doRefuel` method (like refuel operation takes some time)